p2panda-rs/src/test_utils/fixtures/fixtures.rs

```diff
@@ -118,9 +118,9 @@ pub fn delete_message(schema: Hash, #[from(hash)] instance_id: Hash) -> Message
     utils::delete_message(schema, instance_id)
 }
 
-/// Fixture which injects versioned p2panda testing data into a test method.
+/// Fixture which injects p2panda testing data from p2panda version 0.3.0
 #[fixture]
-pub fn v0_2_0_fixture() -> Fixture {
+pub fn v0_3_0_fixture() -> Fixture {
```

Comment:
This version is not released yet but calling that fixture `v0_2_1_fixture` would also be wrong because 0.2.1 doesn't have YASMF hashes.

Reply:
Yes, seems like a good solution.